data/mods/Graphical_Overmap/fonts.json

```diff
@@ -1,6 +1,4 @@
 {
-  "typeface": "unifont",
-  "map_typeface": "unifont",
```

Comment:
Unrelated changes?

Reply:
Ah sorry about that.  I'll rebase it out.  Trying to get that graphical overmap mod working... :fearful: